swagger/src/main/scala/org/http4s/rho/swagger/SwaggerModelsBuilder.scala

```diff
@@ -0,0 +1,261 @@
+package org.http4s
+package rho
+package swagger
+
+import org.http4s.rho.bits.HeaderAST.HeaderRule
+import org.http4s.rho.bits.PathAST._
+import org.http4s.rho.bits.QueryAST.{QueryCapture, QueryRule}
+import org.http4s.rho.bits.ResponseGenerator.EmptyRe
+import org.http4s.rho.bits._
+
+import scala.reflect.runtime.universe.Type
+
+import scalaz._, Scalaz._
+
+class SwaggerModelsBuilder(formats: SwaggerFormats) {
```

Comment:
Agree. Furthermore, we can even make the class SwaggerModelsBuilder private to the package swagger.